hawc/apps/epiv2/models.py

```diff
@@ -0,0 +1,143 @@
+
+    def __str__(self):
+        return self.name
+
+
+class Chemical(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+
+class Criteria(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+
+class MeasurementType(models.Model):
+    description = models.CharField()
+
+    def __str__(self):
+        return self.name
+
+
+class StudyPopulation(models.Model):
+    study = models.ForeignKey(Study, on_delete=models.CASCADE, related_name="study_populations")
+    study_design = models.CharField(choices=constants.StudyDesign.choices, blank=True)
+    source = models.CharField(choices=constants.Source.choices)
+    age_profile = models.ManyToManyField(AgeProfile, blank=True)
+    age_description = models.CharField()
+    sex = models.CharField(default="U", choices=constants.Sex.choices,)
+    summary = models.CharField()
+    countries = models.ManyToManyField(Country, blank=True)
+    region = models.CharField(max_length=128, blank=True)
+    participant_n = models.PositiveIntegerField()
+    criteria = models.ManyToManyField(Criteria, blank=True)
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class Exposure(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="exposures"
+    )
+    measurement_type = models.ManyToManyField(MeasurementType)
+    measurement_timing = models.CharField()
+    exposure_route = models.CharField()
+    analytic_method = models.CharField()
+    comments = models.CharField()
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class ExposureLevel(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="exposure_levels"
+    )
+    chemical = models.ForeignKey(Chemical, on_delete=models.CASCADE)
+    exposure_measurement = models.ForeignKey(Exposure, on_delete=models.CASCADE)
+    sub_population = models.CharField()
+    # TODO: should some of this be separated into another model?
+    central_tendency = models.CharField()
+    central_tendency_type = models.CharField()
+    minimum = models.FloatField()
+    maximum = models.FloatField()
+    percentile25 = models.FloatField()
+    percentile75 = models.FloatField()
+    neg_exposure = models.FloatField()
+    comments = models.CharField()
+    data_location = models.CharField()
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class Outcome(models.Model):
+    study_population = models.ForeignKey(
+        StudyPopulation, on_delete=models.CASCADE, related_name="outcomes"
+    )
+    health_outcome = models.CharField()
+    health_outcome_system = models.CharField()
+    measurement_timing = (
```

Comment:
i agree that we don't need another model, though i think we should put this on the `DataExtraction` table instead, since you may measure the same outcome over multiple time periods